crs_support/urdf/swri_demo.urdf.xacro

```diff
@@ -528,14 +605,13 @@
             -->
             <ros>
               <namespace>custom_ns</namespace>
-              <argument>camera/image_raw:=camera/custom_image</argument>
-              <argument>camera/image_depth:=camera/custom_image_depth</argument>
-              <argument>camera/camera_info:=camera/custom_info_raw</argument>
-              <argument>camera/camera_info_depth:=camera/custom_info_depth</argument>
-              <argument>camera/points:=camera/pointcloud</argument>
+              <argument>custom_camera/image_raw:=custom_camera/custom_image</argument>
+              <argument>custom_camera/image_depth:=custom_camera/custom_image_depth</argument>
+              <argument>custom_camera/camera_info:=custom_camera/custom_info_raw</argument>
+              <argument>custom_camera/camera_info_depth:=custom_camera/custom_info_depth</argument>
+              <argument>custom_camera/points:=custom_camera/custom_points</argument>
```

Comment:
why rename to custom?

Reply:
I just set it up like that originally, I don't know that there is a particular reason

Reply:
These may not map correctly, besides the main application looks for the `image` and `pointcloud` topics during scan so it may fail since those topics have been renamed

Reply:
Good catch. I will change so that it matches the crs.urdf.xacro arguments

Reply:
Fixed